lib/model.js

```diff
@@ -1885,6 +1885,29 @@ class Model {
     return this.findOne(options);
   }
 
+  /**
+ * Search for a many instances by its primary key._
+ *
+ * @param  {Array}      param The value of the desired instance's primary key.
+ *
+ * @see
+ * {@link Model.findAll}           for a full explanation of options, Note that options.where is not supported.
+ *
+ * @returns {Promise<Model>}
+ */
```

Comment:
Please fix indentation here (two extra spaces per line needed) and remove these big blank spaces like here:

```
@param  {Array}      param
```

Also the method returns `Promise<Array<Model>>`